domain-models/samples/AU128.tern.ttl

```diff
@@ -0,0 +1,13 @@
+@prefix tern: <https://w3id.org/tern/ontologies/tern/> .
```

Comment:
TERN ontology has a validator which I've loaded into our [RDFtools tool's validation page](http://rdftools.surroundaustralia.com/validate). We won't perhaps get these samples to pass validation since the TERN ont's looking for lots more info than we have, but we might get close and should try.